.travis.yml

```diff
@@ -10,20 +10,20 @@ env:
   global:
   - DOCKER_REPO=$TRAVIS_REPO_SLUG
 before_install:
-- apt-cache madison docker-engine
-- travis_retry sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine
+- apt-cache madison docker-ce
```

Comment:
@erikdw @JessicaLHartog thanks for all the shepherding here, really appreciate all the help and effort. I think I must have been a "gitiot" here, after I rebased, I saw a message that my master was a few commits behind, I assumed there were changes in my own master that I missed from the remote and did a pull, triggering the merge... It seems like we are OK though, but this is probably what went wrong.